server/sonar-db-migration/src/main/java/org/sonar/server/platform/db/migration/version/v64/CreateMembersGroupsInEachOrganization.java

```diff
@@ -0,0 +1,92 @@
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.server.platform.db.migration.version.v64;
+
+import java.sql.SQLException;
+import java.util.Date;
+import org.sonar.api.utils.System2;
+import org.sonar.db.Database;
+import org.sonar.server.platform.db.migration.step.DataChange;
+import org.sonar.server.platform.db.migration.step.MassUpdate;
+
+public class CreateMembersGroupsInEachOrganization extends DataChange {
+
+  private static final String GROUP_MEMBERS = "members";
+
+  private final System2 system2;
+
+  public CreateMembersGroupsInEachOrganization(Database db, System2 system2) {
+    super(db);
+    this.system2 = system2;
+  }
+
+  @Override
+  protected void execute(Context context) throws SQLException {
+    Date now = new Date(system2.now());
+    if (isOrganizationEnabled(context)) {
+      createMembersGroup(context, now);
+      createPermissionTemplateGroups(context, now);
+    }
+  }
+
+  private static void createMembersGroup(Context context, Date now) throws SQLException {
+    MassUpdate massUpdate = context.prepareMassUpdate().rowPluralName("groups");
+    massUpdate.select("SELECT o.uuid FROM organizations o " +
+      "WHERE NOT EXISTS (SELECT 1 FROM groups g WHERE g.organization_uuid=o.uuid AND g.name=?)").setString(1, GROUP_MEMBERS);
+    massUpdate.update("INSERT INTO groups (organization_uuid, name, description, created_at, updated_at) values (?, ?, ?, ?, ?)");
+    massUpdate.execute((row, update) -> {
+      update
+        .setString(1, row.getString(1))
+        .setString(2, GROUP_MEMBERS)
+        .setString(3, "All members of the organization")
+        .setDate(4, now)
+        .setDate(5, now);
+      return true;
+    });
+  }
+
+  private static void createPermissionTemplateGroups(Context context, Date now) throws SQLException {
+    MassUpdate massUpdate = context.prepareMassUpdate().rowPluralName("permission templates on groups");
+    massUpdate.select("SELECT g.id, pt.id FROM organizations o " +
+      "INNER JOIN permission_templates pt ON pt.kee=o.default_perm_template_project " +
+      "INNER JOiN groups g ON g.organization_uuid=o.uuid AND g.name=? " +
+      "WHERE NOT EXISTS (SELECT 1 FROM perm_templates_groups ptg WHERE ptg.group_id=g.id AND ptg.template_id=pt.id)")
+      .setString(1, GROUP_MEMBERS);
+    massUpdate.update("INSERT INTO perm_templates_groups (group_id, template_id, permission_reference, created_at, updated_at) values (?, ?, ?, ?, ?)");
+    massUpdate.update("INSERT INTO perm_templates_groups (group_id, template_id, permission_reference, created_at, updated_at) values (?, ?, ?, ?, ?)");
```

Comment:
it's on purpose? I thought it was an error. If it's on purpose, can you explain why?